agent/hippo4j-agent-bootstrap/src/main/java/cn/hippo4j/agent/bootstrap/Hippo4jAgent.java

```diff
@@ -17,22 +17,17 @@
 
 package cn.hippo4j.agent.bootstrap;
 
-import cn.hippo4j.agent.core.boot.AgentPackageNotFoundException;
 import cn.hippo4j.agent.core.boot.ServiceManager;
-import cn.hippo4j.agent.core.conf.Config;
 import cn.hippo4j.agent.core.conf.SnifferConfigInitializer;
 import cn.hippo4j.agent.core.jvm.LoadedLibraryCollector;
-import cn.hippo4j.agent.core.logging.api.ILog;
-import cn.hippo4j.agent.core.logging.api.LogManager;
-import cn.hippo4j.agent.core.plugin.AbstractClassEnhancePluginDefine;
-import cn.hippo4j.agent.core.plugin.EnhanceContext;
-import cn.hippo4j.agent.core.plugin.InstrumentDebuggingClass;
-import cn.hippo4j.agent.core.plugin.PluginBootstrap;
-import cn.hippo4j.agent.core.plugin.PluginException;
-import cn.hippo4j.agent.core.plugin.PluginFinder;
+import cn.hippo4j.agent.core.plugin.*;
```

Comment:
Do not import *, please check all submissions.

Reply:
OK，IDAE auto-import package configuration is not set. I will modify it and recommit.